developers/weaviate/client-libraries/typescript/v2_v3_migration.md

```diff
@@ -83,15 +55,15 @@ To go from `v2` to `v3`, you must
 
 ## Instantiation
 
-The `v3` client is instantiated through the `WeaviateClient` object, which is the main entry point for all API operations.
+The `v3` client is instantiated through the `weaviate` object, which is the main entry point for all API operations.
 
 You can directly instantiate the client, but in most cases you can use helper functions starting with `connectTo`, such as `connectToLocal`, `connectToWCS`. 
```

Comment:
Consider

You can instantiate the client directly, but in most cases you should use the helper functions [`connectToLocal`](#NEEDS_LINK), [`connectToWCS`](#NEEDS_LINK), and [`connectToCustom](#NEEDS_LINK).